source/threads.tex

```diff
@@ -1220,6 +1223,16 @@
 equal to the \tcode{thread::id} object of any \tcode{thread} object that
 does not represent threads of execution.
 
+\pnum
+The \defnadj{text}{representation} for
+the character type \tcode{charT} of an object of type \tcode{thread::id}
+is an unspecified sequence of \tcode{charT} such that,
```

Comment:
This isn't actually defining "text representation". It's defining the "text representation for `charT` of a `thread::id`". So I don't think this should appear in the index, given that it's only used in the present subclause.
```suggestion
The \defn{text representation} for
the character type \tcode{charT} of an object of type \tcode{thread::id}
is an unspecified sequence of \tcode{charT} such that,
```

Reply:
ok, done